packages/web-components/@sfx/search-box/src/events.ts

```diff
@@ -0,0 +1,12 @@
+export const SEARCHBOX_EVENT = {
+  UPDATE_SEARCH_TERM: 'sfx::update_search_term',
+  SEARCH_REQUEST: 'sfx::search_request',
+  SEARCHBOX_CLEAR_CLICK: 'sfx::searchbox_clear_click',
+  SEARCHBOX_CLICK: 'sfx::searchbox_click_event',
+  SEARCHBOX_HOVER: 'sfx::searchbox_hover_event',
+  SEARCHBOX_CHANGE: 'sfx::on_searchbox_change'
```

Comment:
I would remove the `on_`. "On" should be for the listeners, but the event describes the actual action.

```suggestion
  SEARCHBOX_CHANGE: 'sfx::searchbox_change'
```